### ISSUE TITLE
Use the unallocated pin on the USBC connector as a PPS input pin

### ISSUE THREAD

**Issue by juchong** [ajn96/iSensor-SPI-Buffer#28]

**Comment by ajn96** [comment]
@juchong good idea! We should be able to use on of the dedicated USB alternate function pins. I'll need to take a look at what pin would be good on the micro. In general before you start the layout on the Rev C we should probably do a full schematic review

**Comment by juchong** [comment]
An additional adapter/injection board will need to be developed to take advantage of this feature.

**Comment by ajn96** [comment]
This has been incorporated into the rev C board design. The USB alternate function line goes into a mux, which can be configured by software to connect the line to any of the IMU DIO lines.

**Comment by ajn96** [comment]
Plan is to add a "SB_MUX" bitfield to CLI_CONFIG to support this mux functionality
